### PR TITLE
Upgrade postcss ^7.0.2 and autoprefixer ^9.0.2

### PR DESCRIPTION
**Pull request by Duder-onomy**
[postcss changelogs](https://github.com/postcss/postcss/blob/master/CHANGELOG.md)
[autoprefixer changelogs](https://github.com/postcss/autoprefixer/blob/master/CHANGELOG.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,9 +9,6 @@
 		"email": "sindresorhus@gmail.com",
 		"url": "sindresorhus.com"
 	},
-	"engines": {
-		"node": ">=6"
-	},
```
Comment (sindresorhus) [inline package.json]: Don't do unrelated changes.

Comment (Duder-onomy) [reply package.json]: Good catch.
That should not have made it in here. I will close this and re-open from a diff SHA. 
No one wants a dirty git history.

**Comment by sindresorhus** [comment]
@Duder-onomy There's no need to open a new PR, you can just push additional commits and I'll squash when merging. Opening a new PR just creates PR noise and is totally unnecessary.

**Comment by Duder-onomy** [comment]
Thanks. We all have our ways.
